package.xml.tpl

```diff
@@ -38,7 +38,7 @@
             <file role="php" baseinstalldir="gherkin" name="vendor/.composer/autoload.php" />
             <file role="php" baseinstalldir="gherkin" name="vendor/.composer/autoload_namespaces.php" />
             <file role="php" baseinstalldir="gherkin" name="phpdoc.ini.dist" />
-            <file role="php" baseinstalldir="gherkin" name="phpunit.xml.dist" />
+            <file role="php" baseinstalldir="gherkin" name="phpunit.dist.xml" />
```

Comment:
Is this package.xml.tpl actually used for anything? It looks quite outdated (minimum php = 5.3.1, for example).

Reply:
I have absolutely no idea. I understood it might relate to PEAR/PECL somehow? I couldn't find reference to it anywhere though.

Same situation with `phpdoc.ini`.

Reply:
This is indeed a left-over from the time this package was available through a PEAR channel years ago.

Reply:
I think we can delete it, also phpdoc.ini.dist, and I can't find any usages of the `libpath.php` either so I'm guessing that might also be PEAR/something-pre-composer-related. Probably best to remove those in a separate PR, I can do that at some point unless you have a chance @uuf6429

Reply:
I'll do all the changes later on today, I can also do that.